Support L5.5, update spatie/laravel-backup, rename config to backup.php

cristianuibar opened the pull request:
Fixes #40

iMokhles commented:
@tabacitu @lloy0076 any update when this will be reviewed and merged ?

gustavobigardi commented:
Is anyone looking at this PR?

lloy0076 commented on src/config/backup.php:
```diff
@@ -0,0 +1,159 @@
+<?php
```
I guess we can publish this but I wonder if upstream publishes this _as well_?

lloy0076 commented on src/config/backup.php:
```diff
@@ -0,0 +1,159 @@
+                 * You can specify individual files as well.
+                 */
+                'exclude' => [
+                    base_path('vendor'),
```
Why not `node_modules` too? See https://docs.spatie.be/laravel-backup/v5/installation-and-setup

gotrecillo replied:
I think we could just not publish the config file ourselves.
spatie package has autodiscovery.

we could just add this line to README.

```
php artisan vendor:publish --provider=" Spatie\Backup\BackupServiceProvider"
```

This way we will not have to mantain our config file to mimic theirs.

What do you think?

lloy0076 replied:
So long as it goes into the documentation for backpack - @tabacitu.

lloy0076 commented:
Nothing I have to say is a major show stopper really - @tabacitu, what say you here?

OTOH this might be a breaking change on an upgrade.

cristianuibar commented:
Fixed conflicts and corrected the Romanian translation. Can this be merged or another fix be applied? So much time has passed and I'm still waiting for a fix, it's getting frustrating.

tabacitu commented:
Excellent PR @cristianuibar . Thanks a lot! The backups didn't work for me with the configuration file you provided, but I pulled the new config file from the spatie/backup package and made the appropriate changes - works out of the box now.

I've published this PR as a new point version (1.3.x) so please use [these steps](https://github.com/Laravel-Backpack/BackupManager#upgrading-from-12x-to-13x) if you need to upgrade your existing projects.

Cheers! Keep 'em coming :-)
